client/client-multi/client-cpp/kaa/event/EventManager.hpp

```diff
@@ -70,13 +70,15 @@ class EventManager : public IEventManager
 
     virtual void setTransport(EventTransport *transport);
 
+    using AbstractTransactable::beginTransaction;
```

Comment:
No, this is just warning fix:
In file included from /Users/vasiliy/Works/CyberVision/Internal/kaa/git-repos/kaa-vnerozin/client/client-multi/client-cpp/impl/KaaClient.cpp:19:
In file included from /Users/vasiliy/Works/CyberVision/Internal/kaa/git-repos/kaa-vnerozin/client/client-multi/client-cpp/test/../kaa/KaaClient.hpp:24:
/Users/vasiliy/Works/CyberVision/Internal/kaa/git-repos/kaa-vnerozin/client/client-multi/client-cpp/test/../kaa/event/EventManager.hpp:73:30: warning: 
      'kaa::EventManager::beginTransaction' hides overloaded virtual function [-Woverloaded-virtual]
    virtual TransactionIdPtr beginTransaction()
                             ^
/Users/vasiliy/Works/CyberVision/Internal/kaa/git-repos/kaa-vnerozin/client/client-multi/client-cpp/test/../kaa/transact/AbstractTransactable.hpp:30:30: note: 
      hidden overloaded virtual function 'kaa::AbstractTransactable<std::__1::list<kaa::Event, std::__1::allocator<kaa::Event> > >::beginTransaction' declared
      here: different number of parameters (1 vs 0)
    virtual TransactionIdPtr beginTransaction(IKaaClientContext &context_) {
                             ^